SKILL.tsv

```diff
@@ -1267,7 +1267,7 @@ SKILL_20150317_001266	{memo X}Buff for adding attack speed
 SKILL_20150317_001267	{memo X}Make ground skill of enemies be usable by allies.
 SKILL_20150317_001268	{memo X}test
 SKILL_20150317_001269	$Multi Shot
-SKILL_20150317_001270	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Quickly launch multiple arrows at a specified location.
+SKILL_20150317_001270	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Quickly launch multiple arrows at an enemy.
```

Comment:
'Quickly launch multiple arrows to an enemy at a specified location' is overly specific. "At an enemy is simpler, or perhaps "Quickly launch multiple arrows forward" is an even simpler translation.